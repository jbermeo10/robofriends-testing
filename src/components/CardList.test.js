import React from 'react';
import { shallow } from 'enzyme';
import Cardlist from './Cardlist';

it('se espera renderizar el componente Cardlist', () => {
  const mockRobots = [
    {
      id: 1,
      name: 'John Snow',
      username: 'JohnJohn',
      email: 'john@gmail.com'
    }
  ]
  expect(shallow(<Cardlist robots={mockRobots}/>)).toMatchSnapshot();
})

