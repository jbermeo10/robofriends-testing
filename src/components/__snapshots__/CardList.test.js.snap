// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`se espera renderizar el componente Cardlist 1`] = `
<div>
  <Card
    email="john@gmail.com"
    id={1}
    key="0"
    name="John Snow"
  />
</div>
`;
